mxd/performance-tests/mxd-performance-evaluation/results_aggregation.py

```diff
@@ -42,8 +42,11 @@ def load_stats(stat_file):
     processes_data = {}
 
     for process_name, process_data in data.items():
-        median_res_time = process_data['medianResTime']
-        processes_data[process_name] = median_res_time
+        median_val = process_data[metric_name]
```

Comment:
Use metric_val instead of median_val, as median already refers to a concrete type of metric.

Reply:
Updated variable name